src/commands/script-dump.json

```diff
@@ -0,0 +1,44 @@
+{
+    "DUMP": {
+        "summary": "Dump server-side Lua scripts in the script cache.",
+        "complexity": "O(N) with N being the number of scripts to check (so checking a single script is an O(1) operation).",
+        "group": "scripting",
+        "since": "7.2.4",
+        "arity": -3,
+        "container": "SCRIPT",
+        "function": "scriptCommand",
+        "command_flags": [
+            "NOSCRIPT"
+        ],
+        "acl_categories": [
+            "SCRIPTING"
+        ],
+        "command_tips": [
+            "REQUEST_POLICY:ALL_SHARDS",
+            "RESPONSE_POLICY:AGG_LOGICAL_AND"
```

Comment:
Probably the wrong command tip. I presume you would want to run this on a single shard, which is the default policy, so this could be omitted.

Reply:
Yes, the command tip is wrong. Thanks